src/main/java/org/jabref/gui/bibtexkeypattern/BibtexKeyPatternPanel.java

```diff
@@ -42,6 +42,10 @@ public BibtexKeyPatternPanel(BasePanel panel) {
         buildGUI();
     }
 
+    public void setDefaultPat(String pattern) { this.defaultPat.setText(pattern); }
```

Comment:
Convert to a property? Seems more logical here

Reply:
In addition, please rename it to `defaultPattern`